coord/src/dist_aggregate.c

```diff
@@ -173,6 +187,7 @@ typedef struct {
   MRIterator *it;
   MRCommand cmd;
   MRCommandGenerator cg;
+  AREQ *areq;
```

Comment:
Not needed if we process the `FORMAT` arg by the coordinator (would not need to read it from the shard)

Reply:
It is still needed since it will not be processed by the coordinator (commented [here](https://github.com/RediSearch/RediSearch/pull/3707/files#r1271432604))